charts/tesk/templates/ingress/ingress-rules.yaml

```diff
@@ -1,23 +1,29 @@
-{{ if .Values.ingress.active }}
-apiVersion: extensions/v1beta1
+{{ if .Values.ingress.rules }}
```

Comment:
Why change the name?

Reply:
I made the change in `values.yaml` to be more explicit. An ingress rule can or cannot be deployed. I had the feeling that `active` was too general (ingress and rules deployments).